File: AM/Lista1/l1z2java/app/src/main/java/com/example/l1z2java/MainActivity.java
package com.example.l1z2java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    private int currentValue = 0;
    private int score = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        rollValue();
    }

    private void rollValue() {
        Random r = new Random();
        currentValue = r.nextInt(100);
    }

    public void checkNumber(View view) {
        String value = ((EditText)findViewById(R.id.textInput)).getText().toString();

        if (value.isEmpty()) return;

        try {
            int compareValue = Integer.parseInt(value);
            if (currentValue == compareValue) {
                Toast.makeText(this, "Dobrzeeeee", Toast.LENGTH_SHORT).show();
                score++;
                rollValue();
            } else if (currentValue > compareValue) {
                ((TextView)findViewById(R.id.hintText)).setText("Podana liczba jest za mała.");
            } else {
                ((TextView)findViewById(R.id.hintText)).setText("Podana liczba jest za duża.");
            }

            ((EditText)findViewById(R.id.textInput)).setText("");
        } catch (NumberFormatException e) {

        }
    }
}

File: AM/Lista1/l1z2kotlin/app/src/main/java/com/example/l1z2_kotlin/MainActivity.kt
package com.example.l1z2_kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import androidx.annotation.IntegerRes
import com.google.android.material.textfield.TextInputLayout
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    private var currentValue = 0
    private var score = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        rollValue()
    }

    private fun rollValue() {
        currentValue = Random.nextInt(100)
    }

    fun checkNumber(view: View) {
        var value = findViewById<EditText>(R.id.textInput).text

        if (value.toString().isEmpty()) return

        try {
            if (currentValue == Integer.parseInt(value.toString())) {
                Toast.makeText(this, "Dobrzeeeee", Toast.LENGTH_SHORT).show()
                score++
                rollValue()
            } else if (currentValue > Integer.parseInt(value.toString())) {
                findViewById<TextView>(R.id.hintText).text = "Podana liczba jest za mała."
            } else {
                findViewById<TextView>(R.id.hintText).text = "Podana liczba jest za duża."
            }

            findViewById<EditText>(R.id.textInput).setText("")
        } catch (e: NumberFormatException) {

        }


    }
}